dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -239,11 +249,15 @@ public void Test_Create_app_descriptor_Then_Create_App_Integration_Then_Delete_T
     @Test
     public void Test_Create_App_descriptor_Then_Create_App_Integration_Then_Delete_One_Single_Secret() {
 
-        final Map<String, Param> paramMap = ImmutableMap.of(
-                "param1", Param.newParam("val-1", false, Type.STRING, "label", "hint", true),
-                "param2", Param.newParam("val-2", false, Type.STRING, "label", "hint", true),
-                "param3", Param.newParam("val-3", false, Type.STRING, "label", "hint", true)
+        final SortedMap<String, ParamDescriptor> paramMap = ImmutableSortedMap.of(
+                "param1", ParamDescriptor
+                        .newParam("default", false, Type.STRING, "label", "hint", true),
+                "param2", ParamDescriptor
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "param2" appears 10 times in this file; the first occurrence is on line 251](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5289044)